src/Stratis.Bitcoin/Configuration/Logging/ILogger.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using Microsoft.Extensions.Logging;
+using Stratis.Bitcoin.Utilities;
+
+namespace Stratis.Bitcoin.Configuration.Logging
+{
+    /*
```

Comment:
This code is commented out, should this be removed?

Reply:
Removed.